### DISCUSSION TITLE
statistics - reports for TOTAL listeners

### DISCUSSION THREAD

**Discussion by marekdziem** [AzuraCast/AzuraCast#6064] [Q&A]
Hi AzuraCats!  @BusterNeece

How can we get reports showing the TOTAL listenership.... i.e. a detailed report showing aggregated listener / connection event history?

for instance:
userA connected at 8am and listened for 15min (1), then again at 10am for 25min (2) and again at 6pm for 75 minutes (3)
userB connected at 11am and listened for 35 minutes (4) and then again at 5pm for 2 hours. (5)

The report would show at TOTAL of 5 listens for total 275 minutes.

Is there a way to expose this metric?

thanks
marek

**Comment by BusterNeece** [comment]
Reports now include total listeners as selectable.

**Comment by IngeEclipse** [comment]
This is fantastic news, thanks so much Buster!

From: Buster Neece ***@***.***>
Reply to: AzuraCast/AzuraCast ***@***.***>
Date: Friday, 17 November 2023 at 04:24
To: AzuraCast/AzuraCast ***@***.***>
Cc: Inge Dziembowski ***@***.***>, Manual ***@***.***>
Subject: Re: [AzuraCast/AzuraCast] statistics - reports for TOTAL listeners (Discussion #6064)


Reports now include total listeners as selectable.
